spec/lib/html_cleaner_spec.rb

```diff
@@ -786,6 +786,39 @@
       expect(doc.xpath(".//br")).to be_empty
     end
 
+    it "does not add paragraphs inside figure" do
+      html = <<~HTML
+        <figure>
+
+          <img src="http://example.com/Camera-icon.svg" alt="camera icon">
+
+          <img src="http://example.com/Hand-icon.svg" alt="hand icon">
+
+        </figure>
+      HTML
+
+      result = add_paragraphs_to_text(html)
+      doc = Nokogiri::HTML.fragment(result)
+
+      expect(doc.xpath("./figure/p")).to be_empty
+    end
+
+    it "keeps figure/figcaption and titles" do
```

Comment:
It looks like we're testing two different things here. Maybe break this into two examples:
- `it "allows alt and title attributes on elements inside figure" do`
- `it "allows other HTML elements inside figcaption" do`

Reply:
Sure, I will do that shortly.